x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/actions/throttle.ts

```diff
@@ -62,13 +66,17 @@ export default ({ getService }: FtrProviderContext) => {
           const { body: hookAction } = await supertest
             .post('/api/actions/action')
             .set('kbn-xsrf', 'true')
+            .set(ELASTIC_HTTP_VERSION_HEADER, '2023-10-31')
+            .set(X_ELASTIC_INTERNAL_ORIGIN_REQUEST, 'kibana')
             .send(getWebHookAction())
             .expect(200);
 
           const rule = await createRule(supertest, log, getRuleWithWebHookAction(hookAction.id));
           const {
             body: { mute_all: muteAll, notify_when: notifyWhen, actions },
-          } = await supertest.get(`/api/alerting/rule/${rule.id}`);
+          } = await supertest
+            .get(`/api/alerting/rule/${rule.id}`)
+            .set(X_ELASTIC_INTERNAL_ORIGIN_REQUEST, 'kibana');
```

Comment:
Just curious if we really need here this header, my understanding was that it's needed only for internal requests

Reply:
That's correct since the `notify_when` property is internal we need this header to access it!